src/Features/CSharp/Portable/QuickInfo/CSharpDiagnosticAnalyzerQuickInfoProvider.cs

```diff
@@ -25,17 +25,17 @@ namespace Microsoft.CodeAnalysis.CSharp.QuickInfo;
 [ExtensionOrder(Before = QuickInfoProviderNames.Semantic)]
 [method: ImportingConstructor]
 [method: SuppressMessage("RoslynDiagnosticsReliability", "RS0033:Importing constructor should be [Obsolete]", Justification = "Used in test code: https://github.com/dotnet/roslyn/issues/42814")]
-internal sealed class CSharpDiagnosticAnalyzerQuickInfoProvider(DiagnosticAnalyzerInfoCache.SharedGlobalCache globalCache) : CommonQuickInfoProvider
```

Comment:
end goal is to make this type  no longer exported.  it will simply be a helper type only for use by IDiagAnalService.